lib/iron_bank/action.rb

```diff
@@ -30,6 +33,18 @@ def name
       self.class.name.split('::').last
     end
 
+    def success?
+      object.fetch(:success, true)
+    end
+
+    def object
+      @object ||= ::IronBank::Object.new(body.first).deep_underscore
```

Comment:
Why use `body.first` here? Is there something specific in the Zuora response or is it expected that every `body` returns an `Enumerable`?

Reply:
All Zuora actions return an array
Ref: https://www.zuora.com/developer/api-reference/#operation/Action_POSTamend

Reply:
The query action returns a different response, will cover that condition too.